src/components/calcite-input/calcite-input.tsx

```diff
@@ -349,6 +360,13 @@ export class CalciteInput {
   //
   //--------------------------------------------------------------------------
 
+  @Listen("click")
```

Comment:
I think the JSX Prop is actually preferred according to linting.

Reply:
For some reason without this `@Listen`er the click event wasn't triggering consistently.  I'll try adding the click handler back in the JSX to see if that works like it should.

Reply:
This click handler isn't needed anymore.  It is working fine without it, so this must have been an old change I thought I needed but forgot to remove.